projects/adf4368/src.mk

```diff
@@ -0,0 +1,45 @@
+include $(PROJECT)/src/platform/$(PLATFORM)/platform_src.mk
+include $(PROJECT)/src/examples/examples_src.mk
+
+SRCS += $(PROJECT)/src/platform/$(PLATFORM)/main.c
+
+INCS += $(PROJECT)/src/common/common_data.h
+SRCS += $(PROJECT)/src/common/common_data.c
+
+INCS += $(PROJECT)/src/platform/platform_includes.h
+
+INCS += $(PROJECT)/src/platform/$(PLATFORM)/parameters.h
+SRCS += $(PROJECT)/src/platform/$(PLATFORM)/parameters.c
+
+SRCS += $(DRIVERS)/api/no_os_uart.c     \
+        $(DRIVERS)/api/no_os_gpio.c     \
+	$(DRIVERS)/api/no_os_spi.c  	\
+	$(DRIVERS)/api/no_os_irq.c 	\
+	$(DRIVERS)/api/no_os_timer.c	\
+        $(DRIVERS)/api/no_os_dma.c      \
+        $(NO-OS)/util/no_os_fifo.c      \
+	$(NO-OS)/util/no_os_mutex.c     \
+	$(NO-OS)/util/no_os_lf256fifo.c \
+        $(NO-OS)/util/no_os_list.c      \
+        $(NO-OS)/util/no_os_util.c      \
+        $(NO-OS)/util/no_os_alloc.c
+
+INCS += $(INCLUDE)/no_os_delay.h     \
```

Comment:
you could use `NO_OS_INCS_DIR += $(INCLUDE)` to replace everything from line 27 to 42

Reply:
Fixed

Reply:
we discussed in private this is not applicable here because there is a potentially a bug about NO_OS_INC_DIRS on mbed platform